services/app-api/handlers/reports/release.ts

```diff
@@ -0,0 +1,184 @@
+import handler from "../handler-lib";
+// utils
+import dynamoDb from "../../utils/dynamo/dynamodb-lib";
+import {
+  DynamoGet,
+  DynamoWrite,
+  isMLRReportMetadata,
```

Comment:
will it only be MLR that can use this functionality? or will NAAAR reports get locked/unlocked as well?

EIther way we could use interface inheritance to standardize this method